jupyter_server/services/contents/largefilemanager.py

```diff
@@ -91,7 +92,8 @@ async def save(self, model, path=""):
 
             os_path = self._get_os_path(path)
             self.log.debug("Saving %s", os_path)
```

Comment:
Seems like this debug statement (and same with the synchronous version above), should either be moved within the `chunk == 1` block or (better yet, IMHO), include the chunk number.  Perhaps something like:
```python
self.log.debug(f"Saving chunk {chunk} of file {os_path}")
```